src/rust/ide/src/double_representation/module.rs

```diff
@@ -598,18 +608,27 @@ pub fn locate
 ///
 /// The module is assumed to be in the file identified by the `method.file` (for the purpose of
 /// desugaring implicit extensions methods for modules).
+///
+/// The `name` parameter is the name of the module that contains `ast`. It affects how the `here`
+/// keyword is resolved.
 pub fn lookup_method
-(ast:&known::Module, method:&language_server::MethodPointer) -> FallibleResult<definition::Id> {
-    let module_name                    = QualifiedName::try_from(method)?;
-    let explicitly_extends_looked_type = method.defined_on_type == module_name.name().as_ref();
-
+(name:&ReferentName, ast:&known::Module, method:&language_server::MethodPointer)
```

Comment:
I would rename `name` parameter to `module_name`

Reply:
done